### PR TITLE
make script compatible with volume output

### PR DESCRIPTION
**Pull request by NicoSchlw**

### CODE REVIEW

**Comment by krenzland** [comment]
What exactly is this computing? What is the GME of a volume output?

**Comment by NicoSchlw** [comment]
It might be interesting to use some of the (low-frequency) metrics of the script also for volume output. For example, I'm comparing how peak displacements and peak velocities decay with distance at the surface and at depth.

**Comment by NicoSchlw** [comment]
I updated the PR based on the reviews, I liked Thomas's idea of using the shape of connect to identify volume data

File: postprocessing/science/GroundMotionParametersMaps/ComputeGroundMotionParametersFromSurfaceOutput_Hybrid.py [hunk 1]
```diff
@@ -308,12 +321,12 @@ def ComputeGroundMotionParameters(args2):
      os.remove(fname)
 
   # Copy connect and geometry:
```
[the remark this answers is not in the document]
Comment (NicoSchlw) [reply postprocessing/science/GroundMotionParametersMaps/ComputeGroundMotionParametersFromSurfaceOutput_Hybrid.py]: Yes good idea, I did it now

File: postprocessing/science/GroundMotionParametersMaps/ComputeGroundMotionParametersFromSurfaceOutput_Hybrid.py [hunk 2]
```diff
@@ -282,73 +292,18 @@ def ComputeGroundMotionParameters(args2):
...
-prefix=fn.split('-')[-2]
+prefix = fn.split('-')[-2] if not isVolumeData else fn.split('.')[-2]
 
 sLowPass=f'_lp{args.lowpass[0]:.1f}' if args.lowpass else ''
 prefixGME = f'{prefix}{sLowPass}-GME'
 
 if not args.noMPI:
-   comm.Barrier()
+    comm.Barrier()
+    myres = comm.gather(myres, root=0)
```
Comment (Thomas-Ulrich) [inline postprocessing/science/GroundMotionParametersMaps/ComputeGroundMotionParametersFromSurfaceOutput_Hybrid.py:302]: oh man, my code was a shame :-)
You can remove also above:
```
h5f = h5py.File('temp'+str(irank)+'.h5','w')
for i,dname in enumerate(dataName):
     hdname = "mesh0/"+dname
     h5f.create_dataset(hdname, (nElements_i,), dtype='d')
     h5f[hdname][:] = myres[:,i]
h5f.close()
```

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
Thank you, that's much better.

**Review by Thomas-Ulrich — APPROVED** [review]
thank you

**Comment by NicoSchlw** [comment]
Hi @krenzland, can you approve this PR?

**Review by krenzland — APPROVED** [review]

**Comment by krenzland** [comment]
Sorry, I wasn't aware that I was blocking this.

**Pull request merged**